packages/app/src/overview/components/update-os-modals/update-available-modal/caution-section-container.component.tsx

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import React from "react"
+import styled from "styled-components"
+import { defineMessages } from "react-intl"
+import { FunctionComponent } from "App/__deprecated__/renderer/types/function-component.interface"
+import Text, {
+  TextDisplayStyle,
+} from "App/__deprecated__/renderer/components/core/text/text.component"
+
+const messages = defineMessages({
+  updateAvailableCautionSectionTitle: {
+    id: "module.overview.updateAvailableCautionSectionTitle",
+  },
+  updateAvailableCautionSectionSequenceUpdateDescription: {
+    id: "module.overview.updateAvailableCautionSectionSequenceUpdateDescription",
+  },
+  updateAvailableCautionSectionSingleUpdateDescription: {
+    id: "module.overview.updateAvailableCautionSectionSingleUpdateDescription",
+  },
+})
+
+interface CautionSectionProps {
+  isSingleRelease: boolean
+}
+
+const CautionSectionContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  margin-top: 2.4rem;
+
+  p {
+    text-align: left;
+  }
+`
```

Comment:
Should be exported to separated files

Reply:
I'd prefer to make this option optional for small components like this one. What do you think? 🤔